features/deleting.feature

```diff
@@ -0,0 +1,22 @@
+Feature: As a visitor,
+  I would like to delete emails,
+  that are receiven
+
+  Background:
+    Given there is users named:
+      | name      | email            | password |
+      | Test-User | email@random.com | password |
+      | User1     | user1@random.com | password |
+
+  Scenario: Deleting a message
+    Given "Test-User" is logged in
+    And I send an email to "User1"
+    And I should be on the "index" page
+    And I click "Logout"
+    Given "User1" is logged in
```

Comment:
Avoid using `Given` when an `And` should do. Also, I understand that you are reusing a step definition (perfectly fine) but `"User1" is logged in` doesn't feel right here. Can you rephrase?